impl_core/src/main/java/io/opencensus/implcore/trace/propagation/B3Format.java

```diff
@@ -81,15 +83,27 @@
       throws SpanContextParseException {
     checkNotNull(carrier, "carrier");
     checkNotNull(getter, "getter");
+
     try {
+      // check for single header format
+      final String b3Header = getter.get(carrier, B3);
+      if (b3Header != null) {
+        return parseB3Value(b3Header);
+      }
+      final String traceStateHeader = getter.get(carrier, TRACESTATE);
+      if (traceStateHeader != null) {
+        final String[] components = traceStateHeader.split("=", 2);
```

Comment:
if there are multiple state entries.. would limit of 2 work?

Reply:
Ah, it looks like I should split on commas first before looking for the b3 entry ([spec for my reference](https://www.w3.org/TR/trace-context/#tracestate-header)). However, based on the discussion in #1400, it seems I should remove the tracestate handling logic altogether.